plugin/adsk/plugin/plugin.cpp

```diff
@@ -315,6 +320,13 @@ MStatus initializePlugin(MObject obj)
         }
     }
 
+#if defined(WANT_UFE_BUILD)
+    MayaUsd::LayerManager::addSupportForNodeType(MAYAUSD_NS::ProxyShape::typeId);
```

Comment:
It looks like right now we're only using the new LayerManager for the autodesk proxy-shape.  The AL one already has it's own version of this, which you didn't want to worry about unifying... but what about the Pixar proxy shape?  Are there any reasons you know of why the Pixar proxy shape wouldn't work, or were you just deferring that decision to Pixar?

@mattyjams, any thoughts on whether you think this would work for the Pixar proxy?  Ideally, I like bringing all the proxy shapes closer together, and adding this to the Pixar too would help with that larger goal...

Reply:
`pxrUsdProxyShape` effectively does **not** support UFE. It's only valid use anymore as far as I'm concerned is for anyone continuing to use `pxrUsdReferenceAssembly`, where activating the "Collapsed" or "Expanded" representation of the  assembly node brings in transient `pxrUsdProxyShape` nodes. The assembly node (poorly) supports edits using the Maya assembly framework instead of UFE, so I wouldn't really consider any workflow involving either of the pxr nodes UFE-compatible.

We're actively working towards deprecating use of our own nodes (and indeed everything under `plugin/pxr`) and switching over to `mayaUsdProxyShape` instead, and I'd advocate for others to do the same.

Reply:
Ok, makes sense!